options.css

```diff
@@ -1,21 +1,110 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
 .container {
-  text-align: center;
+	text-align: center;
+	height: 100vh;
+	width: 100vw;
+	background-color: #ebca00;
+}
+.header {
+	padding: 30px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	background-color: #000;
 }
 
+.header img {
+	margin-right: 10px;
+}
+a {
+	text-decoration: none;
+}
 #saved-urls {
-  width: 650px;
-  display: flex;
-  flex-direction: column;
-  margin: 0 auto;
-}
-
-#saved-urls div {
-  padding: 10px;
-  margin: 0px;
-  margin-top: 10px;
-  text-align: left;
-  border: 1px solid #f0f0f0;
-  display: flex;
-  justify-content: space-between;
-  align-items: center;
+	width: 90vw;
+	display: flex;
+	flex-direction: column;
+	margin: 0 auto;
+	/* background: #fff; */
+	border-radius: 25px;
+	transition: all 0.3s ease;
+}
+#saved-urls img {
+	margin-right: 10px;
+}
+.bookmark:hover {
+	transform: scale(1.07);
+}
+
+.bookmark {
+	padding: 10px;
+	margin: 0px;
+	margin-top: 10px;
+	text-align: left;
+	border: 3px solid #000;
+	border-radius: 25px;
+	cursor: pointer;
+	display: flex;
+	transition: all 0.3s ease;
+	justify-content: flex-start;
+	align-items: center;
+	background-color: #000;
+	text-decoration: none;
+	color: #fff;
+	font-weight: bolder;
+	/* min-width: 250px; */
+}
+
+.bookmark p {
+	overflow: hidden;
+	white-space: nowrap;
+	text-overflow: ellipsis;
+	font-family: cursive;
```

Comment:
The cursive font looks a bit weird can you please remove it.
![image](https://user-images.githubusercontent.com/21277179/66516244-3bf4c300-eafe-11e9-883a-9b0aa6dd913c.png)